client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -830,7 +830,8 @@ export default function CheckoutMainContent( {
 
 	return (
 		<StepContainerV2CheckoutFixer isLargeViewport={ isLargeViewport }>
-			<Step.FullWidthLayout
+			<Step.WideLayout
+				width="xhuge"
```

Comment:
Asked about this `xhuge` variation here: p1743511685985719/1743044978.661229-slack-C08HKNPL2BG
I still prefer the `wide` one:

![image](https://github.com/user-attachments/assets/fd79f28b-9db1-498d-a925-0eaa0a33fcd9)